src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,29 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                .getInstance(ManufacturerDao.class);
+        Manufacturer volvo = new Manufacturer();
+        volvo.setName("Volvo");
+        volvo.setCountry("China");
+        System.out.println("Creating new manufacturer Volvo" + manufacturerDao.create(volvo));
+        System.out.println("Getting one of existing manufacturers from DB "
+                + manufacturerDao.get(3L));
+        System.out.println("Deleting all one of existing manufacturers from DB "
+                + manufacturerDao.delete(8L));
```

Comment:
avoid magic numbers, better using existing models
```suggestion
        System.out.println("Getting one of existing manufacturers from DB "
                + manufacturerDao.get(volvo.getId()));
        System.out.println("Deleting all one of existing manufacturers from DB "
                + manufacturerDao.delete(volvo.getId()));
```